packages/modules/web_themes/koala/source/src/components/ChargePointEcoSettings.vue

```diff
@@ -136,6 +147,12 @@ const current = computed(() =>
   mqttStore.chargePointConnectedVehicleEcoChargeCurrent(props.chargePointId),
 );
 
+const dcCharging = computed(() => mqttStore.DCChargingEnabled);
```

Comment:
"DC" sollte als ein Wort behandelt werden (wurde auch bei der Bezeichnung "dcCharging" so gemacht) und demnach in CamelCase "Dc" heißen. 
```suggestion
const dcCharging = computed(() => mqttStore.DcChargingEnabled);
```
Analog für die anderen DC-Methoden im MQTT Store.